src/components/Image/types.ts

```diff
@@ -0,0 +1,38 @@
+import { TPAComponentProps } from '../../types';
+
+export interface ImageProps extends TPAComponentProps {
+  /** The source could be any absolute full URL or a relative URI of a media platform item */
+  src?: string;
+  /** The intrinsic width of the image in pixels */
+  width?: number;
+  /** The intrinsic height of the image in pixels */
+  height?: number;
+  /** The alternative text description of the image. Allowing better SEO when the image has meaning of content */
+  alt?: string;
+  /** A callback to be called when the image is loaded */
+  onLoad?: React.EventHandler<React.SyntheticEvent>;
+  /** A callback to be called if error occurs while loading */
+  onError?: React.EventHandler<React.SyntheticEvent>;
+  /** Specifies how the image is resized to fit its container */
+  resize?: 'contain' | 'cover';
+  // Specifies the proportional relationship between width and height
+  aspectRatio?: 'square' | 'cinema' | 'landscape' | number;
+  /** An experience to set while the image is fetched and loaded  */
+  loadingBehavior?: 'none' | 'blur';
```

Comment:
I know this isn't part of the PR itself, but the convention in the library is to use enums, so I'm wondering if it's better to align this component to the convention.
Personally I prefer using enums, cause I think it's a great tool for consumers.
In this case this would be:
```jsx
  loadingBehavior?: LoadingBehavior;
  ...
const enum LoadingBehavior {
  none = 'none',
  blur = 'blur'
}
```

Reply:
Same concerns as I mentioned above